example/simple/src/main.c

```diff
@@ -208,6 +208,7 @@ int initialize_core(ocf_cache_t cache, ocf_core_t *core)
 	/* Core configuration */
 	ocf_mngt_core_config_set_default(&core_cfg);
 	core_cfg.name = "core1";
+	core_cfg.name_len = env_strnlen(core_cfg.name, OCF_CORE_NAME_SIZE);
```

Comment:
... same here - name_len should be set to the size of string memory buffer, not the actual string length. Let's discuss live

Reply:
fixed and looking for discussion